git_repo/services/ext/gitbucket.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import logging
+log = logging.getLogger('git_repo.gitbucket')
+
+from ..service import register_target
+from .github import GithubService
+
+import github3
+
+@register_target('bucket', 'gitbucket')
+class GitbucketService(GithubService):
+    fqdn = "localhost"
+    port = 8080
+    def __init__(self, *args, **kwarg):
+        super(GitbucketService, self).__init__(*args, **kwarg)
+
+    def connect(self):
+        self.gh = github3.GitHubEnterprise(self.build_url(self))
```

Comment:
I try it. but it doesn't load fqdn/port from configuration on this phase. Because, load_configuration is called by `RepositoryService.__init__`. And `RepositoryService.__init__` also calls `self.connect`, so self.gh must be initialized before __init__ called.
@guyzmo do you have any idea?